package.json

```diff
@@ -30,6 +30,7 @@
     "@commitlint/cli": "17.7.1",
     "@commitlint/config-conventional": "17.7.0",
     "@ls-lint/ls-lint": "2.0.1",
+    "@types/lodash.snakecase": "4.1.7",
```

Comment:
Can't find where dependency is installed, and I guess it should be for backend workspace

Reply:
lodash.snakecase is installed in common node modules, should I move it to the backend workspace, as well as @types/lodash.snakecase?

Reply:
Diff doesn't show package installed anywhere + yep, it should be part of backend deps, as it's used on backend